config/manifests/inferencepool-resources.yaml

```diff
@@ -80,6 +82,95 @@ spec:
             service: inference-extension
           initialDelaySeconds: 5
           periodSeconds: 10
+        volumeMounts:
+        - name: plugins-config-volume
+          mountPath: "/config"
+      volumes:
+      - name: plugins-config-volume
+        configMap:
+          name: plugins-config
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins-config
+  namespace: default
+data:
+  scheduler-default.yaml: |
```

Comment:
nit: It is more than a scheduler configuration, this the configuration of the whole epp, should we remove the scheduler prefix?

Reply:
maybe call it plugins-config?